SpeechResponder/CustomFunctions/ShipCallsign.cs

```diff
@@ -17,36 +20,87 @@ public class ShipCallsign : ICustomFunction
         public string description => Properties.CustomFunctions_Untranslated.ShipCallsign;
         public NativeFunction function => new NativeFunction((values) =>
         {
-            int? localId = (values.Count == 0 ? (int?)null : (int)values[0].AsNumber);
+            // The game provides three options for callsigns used by in-game ATC:
```

Comment:
LEt's get some test coverage on this, as it's quite complex.

Reply:
Done.